internal/controller/gateway/gatewayclass_controller_test.go

```diff
@@ -54,7 +53,7 @@ var _ = Describe("GatewayClass controller", func() {
 	)
 
 	BeforeEach(func() {
-		ctx := context.Background()
+		ctx := GinkgoT().Context()
```

Comment:
OOC does this do anything different functionaly or is it more of a semantics thing?